cms/core/tests/test_tags.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+
+from cms.core.templatetags.util_tags import extend
+
+
+class AppendTagTest(TestCase):
+    def test_append(self):
+        """Test the original list is modified in place."""
+        series = [{"name": "Series 1"}, {"name": "Series 2"}]
+        series_item = {"name": "Series 3"}
+        extend(series, series_item)
+        self.assertEqual([{"name": "Series 1"}, {"name": "Series 2"}, {"name": "Series 3"}], series)
```

Comment:
nitpick: `self.assertListEqual`

Reply:
To be honest, it was something we've been doing (although it may have come from a different project and I am muddling things up). Thank you for linking to the relevant Python docs.